### ISSUE TITLE
Failed to run Yarn --frozen-lockfile

### ISSUE THREAD

**Issue by SlavchoSpasenoski** [getredash/redash#5980]
I am trying to run redash using this development guide : https://redash.io/help/open-source/dev-guide/setup

But I have issue with **yarn --frozen-lockfile** command.

I have had install all dependencies, requirements but this happens on Redash v10.0.0 and Redash v10.1.0, only works with Master-repo (v11.0.0-dev).

Tried with node version 12.0.0 and 14.17.2

Environment: Ubuntu 20.04

Python 3.8.10

Here is screenshot from error:

![issue](https://user-images.githubusercontent.com/28984924/231006877-16343c77-0afe-464e-bc1d-2c73babe2460.png)

Is there any solution?

**Comment by justinclift** [comment]
Oh, I just remembered something else.  Which version of Yarn are you using?

**Comment by SlavchoSpasenoski** [comment]
Same as described in https://redash.io/help/open-source/dev-guide/setup

npm install --global yarn@1.22.10

But also I tried with npm install --global yarn@1.22.19

Additional, I tried with npm versions 6.13.6, 7.x.x, 9.x.x

still no success...

**Comment by SlavchoSpasenoski** [comment]
weird..

There is not recognized any .git repository, at any level directory

![image](https://user-images.githubusercontent.com/28984924/231707774-ddf1934d-0909-481f-a147-52048faaed80.png)

**Comment by symball** [comment]
Getting started with Redash leads to unnecessary issues. I am considering using it as a BI tool and putting it under review:

* I can see recent commits and a lot of love for this project
* The available distributions are 4 years old
* The developer instructions don't seem to work and are for very old versions of Python / Node

Is this project abandoned? It feels that way. As an alternative, I tested out Apache Superset and had a relatively incredible experience so far

**Comment by justinclift** [comment]
@symball The project has recently moved to becoming a Community Led effort, so we're still sorting a few things out.

**Comment by justinclift** [comment]
@SlavchoSpasenoski I finally got around to writing up a new "How to set up your development environment" page today on the wiki, that works for the current Redash:

https://github.com/getredash/redash/wiki/Local-development-setup

It's been written while using an Ubuntu 20.04 desktop VM, and I've rolled back the VM and gone through these steps several times now (refining things as I go).  So it's in a "known good" state for the current state of things.

That being said, I'm still adding more stuff to the end of that document for reference purposes.  So don't be overly worried about a bunch of TODO items at the end of it. :smile:
